crates/rolldown/src/ecmascript/ecma_generator.rs

```diff
@@ -73,11 +73,22 @@ impl Generator for EcmaGenerator {
         .await?
     };
 
-    let footer = match ctx.options.footer.as_ref() {
-      Some(footer) => footer.call(&rendered_chunk).await?,
-      None => None,
+    let footer = {
+      let footer = match ctx.options.footer.as_ref() {
+        Some(footer) => footer.call(&rendered_chunk).await?,
+        None => None,
+      };
+      ctx
+        .plugin_driver
+        .footer(HookFooterArgs { chunk: &rendered_chunk }, footer.unwrap_or_default())
+        .await?
     };
 
+    // let footer = match ctx.options.footer.as_ref() {
```

Comment:
Just remove them. cc @underfin, I think it's Ok.
Thank you a lot!

Reply:
I just removed, it.